src/main/java/com/termos/scooterrental/SpringBootConsoleApplication.java

```diff
@@ -0,0 +1,196 @@
+package com.termos.scooterrental;
+
+import com.termos.scooterrental.model.Rent;
+import com.termos.scooterrental.model.Scooter;
+import com.termos.scooterrental.repository.RentRepository;
+import com.termos.scooterrental.time.TimeUtils;
+import com.termos.scooterrental.model.ScooterDock;
+import com.termos.scooterrental.model.UserAccount;
+import com.termos.scooterrental.repository.ScooterDockRepository;
+import com.termos.scooterrental.repository.ScooterRepository;
+import com.termos.scooterrental.repository.UserAccountRepository;
+import lombok.AllArgsConstructor;
+import lombok.Lombok;
+import org.springframework.boot.Banner;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.System.exit;
+import static java.lang.System.setOut;
+
+@SpringBootApplication
+@AllArgsConstructor
+public class SpringBootConsoleApplication implements CommandLineRunner {
+    ScooterDockRepository scooterDockRepository;
+    ScooterRepository scooterRepository;
+    UserAccountRepository userAccountRepository;
+    RentRepository rentRepository;
+    static Scanner scan = new Scanner(System.in);
```

Comment:
Umieszczanie `Scanner` w składowej klasy jest rzadką praktyką i jako tako powinno być unikane (o ile nie ma ku temu specyficznych wymagań).

Preferowanym rozwiązaniem jest tworzenie obiektu Scanner tam gdzie to potrzebne i wywoływanie metody `.close()` po zakończeniu korzystania.

Również:
Jeśli korzystasz z pola typu `static` zazwyczaj chcesz również dodać `final`.